x/bank/keeper/genesis.go

```diff
@@ -48,10 +53,13 @@ func (k BaseKeeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
 		panic(fmt.Errorf("unable to fetch total supply %v", err))
 	}
 
-	return types.NewGenesisState(
+	rv := types.NewGenesisState(
 		k.GetParams(ctx),
 		k.GetAccountsBalances(ctx),
 		totalSupply,
 		k.GetAllDenomMetaData(ctx),
+		k.GetAllSendEnabledEntries(ctx),
 	)
+	rv.MigrateSendEnabled()
```

Comment:
```suggestion
```

Reply:
This is needed so that, if a genesis file is created and there's somehow still `Params.SendEnabled` entries, they're moved to the correct place.